src/hackerman/TextColor.ts

```diff
@@ -1,16 +1,17 @@
 export function textColor(backgroundColor: string) {
   if (!backgroundColor || 0 === backgroundColor.length) {
+    // console.debug('No backgroundColor provided, returning default')
     return '#000000'
   } else {
     const input = backgroundColor.toString().replace('#', '')
     if (input.length != 6) {
+      // console.debug('Invalid backgroundColor hex code, returning default')
       return '#000000'
     } else {
-      // todo: replace substr with slice perhaps
-      const red = parseInt(input.substr(0, 2), 16)
-      const green = parseInt(input.substr(2, 2), 16)
-      const blue = parseInt(input.substr(4, 2), 16)
-      // console.debug({red, green, blue})
+      const red = parseInt(input.substring(0, 0 + 2), 16)
```

Comment:
can be refactored to input.substring(0, 2) kek

Reply:
I hate JavaScript